wordpress-mobile/WordPress-iOS
# Update cocoapods version to 1.10.1 in Podfile.lock file

Related to: https://github.com/wordpress-mobile/WordPress-iOS/pull/15621

To test:
1. Run `rake dependencies`.
2. Check that `Podfile.lock` file is not modified.

PR submission checklist:
